### ISSUE TITLE
Higher res texture

### ISSUE THREAD

**Issue by jovanovski** [zesage/flutter_cube#4]
I took the planet example, and provided a higher res picture for the texture, but when I zoom in, the same pixelation occurs on the same level. What am I doing wrong?

**Comment by zesage** [comment]
I found that the encodeImageFromPixels function in some devices will return an image size smaller than you specified, this function in mesh.dart's packingTexture. I'm not sure if this is the reason.

**Comment by jovanovski** [comment]
Hmm could be... I guess this isn't the best package meant to use if you need really high detailed textures? (possibly memory issues)

**Comment by zesage** [comment]
Right. The packingTexture will pack all to one raw texture. What is the resolution of your texture? Try to remove the "_stars" object to take a look.
